src/shell/components/shell-view.scss

```diff
@@ -57,9 +62,9 @@ h1, h2, h3, h4, h5, h6 {
 
 
 //****** TABLE *******
-$table-border-color: #dbdbdb;
+$table-border-color: #ccc;
 $table-border-width: 1px;
-$table-background-alt-color: #fafafa;
+$table-background-alt-color: #f9f9f9;
```

Comment:
Yes, based on the latest redlines.  He updated some of the table-related colors across the board.